src/org/zaproxy/zap/extension/httpsinfo/ExtensionHttpsInfo.java

```diff
@@ -15,45 +15,49 @@
  * See the License for the specific language governing permissions and 
  * limitations under the License. 
  */
-package org.zaproxy.zap.extension.httpsInfo;
+package org.zaproxy.zap.extension.httpsinfo;
```

Comment:
Any objection to allow to override the ID of an add-on when building it? This would allow us to keep the package in lower case and use the old ID.